src/dotnet/commands/dotnet-help/HelpUsageText.cs

```diff
@@ -9,10 +9,10 @@ internal static class HelpUsageText
 {LocalizableStrings.ExecutionUsageDescription}
 
 runtime-options:
-  --additionalprobingpath <path>     {LocalizableStrings.AdditionalprobingpathDefinition}
-  --additional-deps <path>           {LocalizableStrings.AdditionalDeps}
-  --fx-version <version>             {LocalizableStrings.FxVersionDefinition}
-  --roll-forward-on-no-candidate-fx  {LocalizableStrings.RollForwardOnNoCandidateFxDefinition}
+  --additionalprobingpath <path>         {LocalizableStrings.AdditionalprobingpathDefinition}
+  --additional-deps <path>               {LocalizableStrings.AdditionalDeps}
+  --fx-version <version>                 {LocalizableStrings.FxVersionDefinition}
+  --roll-forward-on-no-candidate-fx <n>  {LocalizableStrings.RollForwardOnNoCandidateFxDefinition}
```

Comment:
n? should this say true|false or something like that?

cc @KathleenDollard

Reply:
The value is a number - 0, 1, 2 (basically an enum).
I also wanted something which is short, since otherwise the formatting of the usage text would be a challenge.